zipline/finance/blotter.py

```diff
@@ -329,20 +329,23 @@ def get_transactions(self, bar_data):
                     if not order.open:
                         closed_orders.append(order)
 
+        self.prune_orders(closed_orders)
```

Comment:
Since this was a refactoring to `get_transactions`, should we have test coverage for that method?